tools/cli/util.go

```diff
@@ -909,3 +906,31 @@ func prompt(msg string, autoConfirm bool) {
 		os.Exit(0)
 	}
 }
+func payloadsToString(ps *commonpb.Payloads) string {
+	if ps == nil {
+		return ""
+	}
+	var buf bytes.Buffer
+	buf.WriteString("{")
+	for _, payload := range ps.Payloads {
+		buf.WriteString("\"")
+		buf.WriteString(payloadToString(payload))
+		buf.WriteString("\",")
+	}
+	buf.Truncate(buf.Len() - 1) // cut last comma
+	buf.WriteString("}")
+	return buf.String()
+}
+
+func payloadToString(p *commonpb.Payload) string {
+	if p == nil {
+		return "<nil>"
+	}
+	var data string
+	err := payload.Decode(p, &data)
+	if err == nil {
+		return data
+	} else {
+		return string(p.GetData())
```

Comment:
What does this output?

Reply:
It is a very good question. Actually with this implementation only `string` is displayed properly. I fixed it in #377.